Brocfile.js

```diff
@@ -89,12 +89,42 @@ var tasks = {
 
   webpack: function (masterTree) {
     // transform merge module dependencies into one file
-    return webpackify(masterTree, {
+    var options = {
       entry: "./" + fileNames.mainJs + ".js",
       output: {
         filename: dirs.jsDist + "/" + fileNames.mainJsDist + ".js"
+      },
+      module: {
+        loaders: [
+          {
+            test: /\.js$/,
+            loader: "jsx-loader?harmony",
+            exclude: /node_modules/
```

Comment:
We should, though, make sure it is not part of the liniting process. Otherwise we are going to have some trouble with that.